index.d.ts

```diff
@@ -241,13 +243,14 @@ declare module 'react-native-nfc-manager' {
       toHex: (i: any) => any;
       toPrintable: (i: any) => string;
     };
-    isType(record: NdefRecord, tnf: number, type: string): boolean;
+    isType(record: NdefRecord, tnf: TNF, type: string): boolean;
     stringify(data: number[], separator: string): string;
-    encodeMessage(records: NdefRecord[]): Uint8Array;
-    decodeMessage(bytes: any[] | Buffer): NdefRecord[];
+    encodeMessage(records: NdefRecord[]): number[];
```

Comment:
Fixes type incongruence on [writing sample](https://github.com/whitedogg13/react-native-nfc-manager/blob/master/APIv2.md#advanced-usage) in the repo 

![image](https://user-images.githubusercontent.com/11574849/86183194-db803880-bb07-11ea-9b0e-abee7841d19b.png)